CHANGELOG.md

```diff
@@ -73,7 +73,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - docs: add docs page on query performance (close #2316) (#3693)
 - docs: add a sample Caddyfile for Caddy 2 in enable-https section (#4710)
 - docs: add disabling dev mode to production checklist (#4715)
-- docs: add integration guide for AWS Cognito (#4822)
+- docs: add integration guide for AWS Cognito (#4822, #4843)
```

Comment:
I think a changelog was not needed for this particular PR but it's fine cuz it's done.